ui/app/models/secret-engine.js

```diff
@@ -160,13 +160,6 @@ export default class SecretEngineModel extends Model {
     return 'vault.cluster.secrets.backend.list-root';
   }
 
-  get accessor() {
```

Comment:
Duplicate attr removed -- added `v2` back to the template for display rather than attaching to the accessor